ee/codegen/src/__test__/helpers/node-data-factories.ts

```diff
@@ -809,9 +811,11 @@ export function conditionalNodeFactory({
         {
           id: "ad6bcb67-f21b-4af9-8d4b-ac8d3ba297cc",
           rules: [],
-          fieldNodeInputId: "2cb6582e-c329-4952-8598-097830b766c7",
+          fieldNodeInputId:
+            invalidUUID ?? "2cb6582e-c329-4952-8598-097830b766c7",
```

Comment:
nit: I notice that the hard-coded ids are not the same as each other, but that if I provide a value for `invalidUUID`, then that same value is used in multiple places. This makes it seem that these ids are supposed to match up, when in reality (I dont think?) they aren't meant to.

It might be better for tests that care about this case to not use this factory, or to set something higher level, like the `conditions` themselves.